open-authoring/OpenAuthoringErrorInterpreter.ts

```diff
@@ -0,0 +1,122 @@
+import OpenAuthoringError from "./OpenAuthoringError";
+import OpenAuthoringContextualError from "./OpenAuthoringContextualError";
+import { Actions } from "./OpenAuthoringModalContainer";
+import { isForkValid } from "./github/api";
+
+export default async function interpretError(error: OpenAuthoringError) : Promise<OpenAuthoringContextualError> {
+    if (!error || !error.code) {
+        return new OpenAuthoringContextualError(
+            false, // should it be presented as a modal? (if not present a toast)
+            "Error", // title
+            "Failed to interpret error. No context provided.", // message (the only thing a toast will present)
```

Comment:
This message feels a bit more like what a developer should see instead of the error message the user will see. Maybe something more generic?